graphql/service-credentials/index.ts

```diff
@@ -8,14 +8,16 @@ const { ENCRYPTION_SECRET } = process.env
 // They are meant to be set by an operator for display in the /connect page of the UI
 const { DISPLAY_PG_HOSTNAME, DISPLAY_PG_PORT, DISPLAY_PG_DATABASE, DISPLAY_PG_USER } = process.env
 
-type ReplaceGitHubPATInput = {
-  pat: string
+type AddTokenInput = {
+  provider: string
+  type: string
+  token: string
 }
 
 module.exports = makeExtendSchemaPlugin({
   typeDefs: gql`
     extend type Mutation {
-      replaceGitHubPAT(pat: String!): Boolean
+      add_token(provider: UUID!, type: String!, token: String!): Boolean
```

Comment:
we should use camelCase here just to be consistent with the rest of the GraphQL API
```suggestion
      addToken(provider: UUID!, type: String!, token: String!): Boolean
```